cmd/config.go

```diff
@@ -46,29 +46,44 @@ var configurationSetCmd = &cobra.Command{
 	Short: "Set a value in odo config file",
 	Long: `Set an individual value in the Odo configuration file 
 Available Parameters:
-UpdateNotification - Controls if an update notification is shown or not (true or false)`,
+UpdateNotification - Controls if an update notification is shown or not (true or false)
+Timeout            - Timeout (in seconds) for OpenShift server connection check`,
 	Example: `
-   # Set a configuration value
-   odo utils config set UpdateNotification false
+  # Set UpdateNotification off
+  odo utils config set UpdateNotification false
+
+  # Set OpenShift server connection check to 20 seconds
+  odo utils config set timeout 20
+
+  # Set 0 to don't timeout requests for OpenShift server check
+  odo utils config set timeout 0
 	`,
 	Args: func(cmd *cobra.Command, args []string) error {
 		if len(args) < 2 {
-			return fmt.Errorf("Please provide a parameter name and value")
+			checkError(fmt.Errorf("Please provide a parameter name and value"), "")
```

Comment:
Why are we using checkError for this? Seems odd in comparison to using return fmt.Errorf ...